va-gov/pages/sitemap-va.xml

```diff
@@ -0,0 +1,507 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was generated from an excel file provided by VA. Since there are not consistent dev / staging environments for these links, opted not to manipulate this file by environment.